applications/FluidDynamicsApplication/custom_elements/two_fluid_navier_stokes.cpp

```diff
@@ -1959,6 +2231,27 @@ void TwoFluidNavierStokes<TElementData>::CondenseEnrichment(
     }
 }
 
+template <class TElementData>
+void TwoFluidNavierStokes<TElementData>::CondenseEnrichment(
```

Comment:
See my comment about the overload of the  `CondenseEnrichment` method.

Reply:
Changed to:
MerelyCondenseEnrichment

Reply:
Well, I think that `MerelyCondense` and `Condense` are not self-descritive function names. I'd keep `CondenseEnrichment` and rename the other method to `CondenseEnrichmentWithVolumeCheck` or something of the sort.

Reply:
Changed to:
CondenseEnrichmentWithContinuity